### PR TITLE
Show how to display "TruffleRuby" instead of "system" on TravisCI

### PR DESCRIPTION
**Pull request by thbar**
This leverages the [new job naming feature](https://github.com/travis-ci/travis-ci/issues/5898#issuecomment-408825617) available in TravisCI.

### CODE REVIEW

**Comment by thbar** [comment]
@eregon right here!

File: doc/user/standalone-distribution.md [hunk 1]
```diff
@@ -56,7 +56,8 @@ script:
 matrix:
   include:
   - rvm: 2.5.1
-  - rvm: system
+  - name: truffleruby
```
Comment (eregon) [inline doc/user/standalone-distribution.md]: I'm a bit hesitating, but I think `TruffleRuby` (with capitalization) would look slightly nicer when looking at https://travis-ci.org/thbar/kiba/builds/424860029
Could you change it? :)

**Comment by thbar** [comment]
I'll have to review that OCA first (will read it in full when I have enough time for that).

**Comment by eregon** [comment]
@thbar Per https://github.com/oracle/truffleruby/pull/1418#issuecomment-418525775, there is no need to sign the OCA for documentation fixes :smiley: 

> @chrisseaton: We can take simple documentation fixes without you signing the OCA (but you can sign the OCA if you want to).

Sorry for the misleading automated message above.

**Comment by thbar** [comment]
@eregon I've changed the casing (initially based on the `rvm versions` output) as you requested.

**Pull request merged**